src/scripts.js

```diff
@@ -142,6 +142,23 @@ function displayCookbook() {
     searchAllRecipes(currentUser.recipesToCook);
   });
   displayRecipes(currentUser.recipesToCook);
+  showRecipesUserCanCook();
+}
+
+function showRecipesUserCanCook() {
+  const recipeCards = document.querySelectorAll('.recipe-card-js');
+  const recipesUserCanCook = recipeCards.forEach(recipeCard => {
+    const recipeOnCard = currentUser.recipesToCook.find(recipe => {
+      return recipe.id === parseInt(recipeCard.id.slice(2))
+    })
+
+    const haveIngredients = currentUser.pantry.checkPantry(recipeOnCard);
+    if (haveIngredients) {
+      recipeCard.childNodes[3].classList.add('highlight-card');
+    }
+  })
+
+  return recipesUserCanCook;
```

Comment:
This looks great, can't wait to do more testing with it once we can add ingredients to the api!